server/container/guice/cassandra-guice/src/test/java/org/apache/james/DockerCassandraRule.java

```diff
@@ -55,8 +57,13 @@ public void await() {
 
     @Override
     public Module getModule() {
```

Comment:
It's not ok to turn the rule into a stateful object. We'll get into trouble really fast

Reply:
I need to have an aceess to the Cassandra DB ar least at the DAO level in order to simulate the Ghost mailbox bug. I also thought about using directly using the injector.

I will see if I can find a workaround with a CassandraProbe somrething.